src/main/java/de/tum/in/www1/artemis/web/rest/dto/metrics/ExerciseInformationDTO.java

```diff
@@ -5,20 +5,25 @@
 import com.fasterxml.jackson.annotation.JsonInclude;
 
 import de.tum.in.www1.artemis.domain.Exercise;
+import de.tum.in.www1.artemis.domain.enumeration.DifficultyLevel;
+import de.tum.in.www1.artemis.domain.enumeration.ExerciseMode;
 
 /**
  * DTO for exercise information.
  *
- * @param id        the id of the exercise
- * @param shortName shortTitle the short title of the exercise
- * @param title     title the title of the exercise
- * @param start     the start date of the exercise
- * @param due       the due date of the exercise
- * @param maxPoints the maximum achievable points of the exercise
- * @param type      the type of the exercise
+ * @param id           the id of the exercise
+ * @param shortName    shortTitle the short title of the exercise
+ * @param title        title the title of the exercise
+ * @param start        the start date of the exercise
+ * @param due          the due date of the exercise
+ * @param maxPoints    the maximum achievable points of the exercise
+ * @param difficulty   the difficulty level of the exercise
+ * @param exerciseMode the mode of the exercise
+ * @param type         the type of the exercise
  */
 @JsonInclude(JsonInclude.Include.NON_EMPTY)
-public record ExerciseInformationDTO(long id, String shortName, String title, ZonedDateTime start, ZonedDateTime due, Double maxPoints, Class<? extends Exercise> type) {
+public record ExerciseInformationDTO(long id, String shortName, String title, ZonedDateTime start, ZonedDateTime due, Double maxPoints, DifficultyLevel difficulty,
```

Comment:
I just noticed that I need `includedInOverallScore` for the bonus badge, as far as I can see 🤔